samples/bluetooth/central_path_loss_monitoring/src/main.c

```diff
@@ -0,0 +1,197 @@
+#include <bluetooth/scan.h>
+#include <dk_buttons_and_leds.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/logging/log.h>
+
+#define ENABLED 1
+#define DISABLED 0
+#define LOW_PATH_LOSS_LED DK_LED1
+#define MEDIUM_PATH_LOSS_LED DK_LED2
+#define HIGH_PATH_LOSS_LED DK_LED3
+#define HIGH_PATH_LOSS_BORDER 60
+#define LOW_PATH_LOSS_BORDER 40
+#define NUM_ITERATIONS 5
+#define PATH_LOSS_BUFFER_ZONE 5
+#define LED_ON 1
+#define LED_OFF 0
+
+static struct bt_conn *default_conn;
+
+/* Set up logging */
+LOG_MODULE_REGISTER(Central_Unit,
+                    LOG_LEVEL_INF);
+
+static struct bt_uuid_128 bt_uuid =
+  BT_UUID_INIT_128(0x23, 0xd1, 0xbc, 0xea, 0x5f, 0x78, 0x23, 0x15,
```

Comment:
what is this uuid? Is there a reason for searching for this uuid in this sample?

Reply:
As we are not using the LBS, maybe we should find a different way to ensure we connect to the correct device? For example the radio_notification_cb sample using the advertiser name to do this